pkg/backend/path_config.go

```diff
@@ -27,6 +27,7 @@ func (b *backend) configReadOperation(ctx context.Context, req *logical.Request,
 			"provider":         c.Config.ProviderName,
 			"provider_version": c.Config.ProviderVersion,
 			"provider_options": c.Config.ProviderOptions,
+			"refresh_interval": c.Config.RefreshInterval,
```

Comment:
As this is a tuning option I think I'd like to "bury" it a little bit as an advanced feature and make sure people recognize exactly what it's controlling. I also like to make sure people know what unit they're working in so I usually put that in the name of the option. I've found that reduces a lot of confusion.

So I'm thinking of a name like `tune_refresh_check_interval_seconds`/`Tuning.RefreshCheckIntervalSeconds` (where `Tuning` is a new struct in the config), as verbose as that is. What do you think?

Reply:
Ok if that's the way you want to do it.  It makes for some ugly code indents, but it works.  I'll submit that change.